colorscad.sh

```diff
@@ -298,16 +296,16 @@ if [ "$FORMAT" = amf ]; then
 				sed "1,4 d; \$ d; s/<volume>/<volume materialid=\"${id}\">/" "${TEMPDIR}/${COLOR}.amf"
 			else
 				echo "Skipping ${COLOR}!" >&2
-				let SKIPPED++
+				(( SKIPPED++ ))
 			fi
-			let id++
+			(( id++ ))
 			echo -ne "\r  ${id}/${COLOR_COUNT} " >&2
 		done
 		echo '</amf>'
 	} > "$OUTPUT"
 
 	# Strip original current dir prefix, if present, to make message smaller
-	OUT=${OUTPUT#${ORIGINAL_PWD}/}
+	OUT="${OUTPUT#"${ORIGINAL_PWD}"/}"
```

Comment:
The outer quotes aren't needed
```suggestion
	OUT=${OUTPUT#"${ORIGINAL_PWD}"/}
```